### ISSUE TITLE
PNG Export doesn't work when it gets too long PDF might be the solution

### ISSUE THREAD

**Issue by smfrue** [felixbade/assistant#89]

**Comment by felixbade** [comment]
True. I think it is an issue with the `html2canvas` library. However, Assistant should say that this is too long and can’t be exported as a PNG because of that.

As a workaround you can export as markdown.
